### ISSUE TITLE
Dragging app tab onto panel opens overview

### ISSUE THREAD

**Issue by mystery-moss** [home-sweet-gnome/dash-to-panel#2087]
**Checks**
- Confirm that the problem persists when Dash-to-Panel is the only enabled extension ✔️
- `journalctl /usr/bin/gnome-shell -f -o cat &` shows no errors when behavior occurs ✔️
- Search existing opened and closed issues ✔️

**Description**
In some apps, dragging a tab over the panel causes the overview menu to open. I observe this with Firefox and gedit, but not with Chromium. I think the issue started with Gnome 46, but can't say for certain.

This is particularly annoying when rearranging tabs in Firefox with the panel at the top of the screen - I regularly trigger it by accidentally straying over the panel. If this is a feature rather than a bug, I'd appreciate an option to disable it!

**Linux distribution and version**
Arch

**GNOME Shell version**
GNOME Shell 46.0

**Dash-to-Panel version**
62

**Where was Dash-to-Panel installed from?**
[Arch repo](https://archlinux.org/packages/extra/any/gnome-shell-extension-dash-to-panel/)

**Screenshots / Video captures**
[Example.webm](https://github.com/home-sweet-gnome/dash-to-panel/assets/29258659/797ef09f-40dc-46e8-8d54-7e367a32814f)

**Comment by charlesg99** [comment]
Thanks, it was an old gnome-shell feature that was removed, so it is now removed from dash to panel. Fixed on master

**Comment by francislavoie** [comment]
@charlesg99 This still happens for me on version 68.

- I use a multi-monitor setup with the dock set on the left edge of the right monitor (so the dock is between both monitors)
- I select some text on the left monitor's Firefox (`firefox-devedition`) and drag it through the dock (anywhere, not even over any app icons and through an empty portion of the dock, not waiting on the dock at all, very rapidly passing it) to drop it in some application on the right monitor
- Overview is triggered _instantly_ as the mouse collides with the dock
- I'm stuck holding my mouse button down, confused, unsure of where to move my mouse now because the target I was looking at is now somewhere unexpected in the overview screen
- I want to either turn off opening the overview _completely_ (I hate it, never use it, I typically bring forward an app I want to drop onto before dragging anyway), or at least set a "hover duration" to trigger it to avoid misfires.

Should this be reopened, or is this a distinct issue?

**Comment by francislavoie** [comment]
@charlesg99 ☝️ Should this be reopened, or should I open a new issue?

**Comment by charlesg99** [comment]
We can reopen it, I guess this is related to #2218.

**Comment by francislavoie** [comment]
This is still a problem. If I have two firefox windows, one open on each monitor side by side, with Dash to Panel in between (on the left edge of the right monitor), if I drag the firefox tab through Dash to Panel (even if I go as fast as possible without sitting on Dash to Panel), it causes overview to open which is extremely annoying.

**Comment by francislavoie** [comment]
Still a thing. I fixed it in my PR. Needs review.
